lib/utils.js

```diff
@@ -205,6 +205,27 @@ utils.getResourceUID = function getResourceUID(namespace, resource) {
         .update(namespace + resource, 'binary').digest('hex');
 };
 
+/**
+ * Validate request headers to determine if readable characters
+ * @param {object} headers - headers attached to the http request (lowercased)
+ * @return {boolean} - returns true if headers are readable, false if not
+ */
+utils.checkReadableHeaders = function checkReadableHeaders(headers) {
```

Comment:
Are you sure the returning value is a boolean ?
For what I understand ```isReadableHeaders``` is an array of valid headerKey. Should you return ```isReadableHeader.length !== 0``` ?